source/guides/administrator.rst

```diff
@@ -115,6 +115,10 @@ Learn how to configure and administer Mattermost apps.
    :glob:
 
    /mobile/mobile-overview.rst
+   /mobile/use-prebuilt-mobile-apps.rst
```

Comment:
High level question. The mobile apps guides are currently buried under the self managed admin guide, does that make sense? Parts of it apply to cloud ie using pre-built apps with HPNS/TPNS and optionally deploying via EMM

Reply:
Agreed -- the mobile content was buried under the Self-Managed Admin Guide. With the new site architecture, the mobile deploy content will live in the Deploy > Mobile Apps section, which is far more intuitive and discoverable.

Reply:
This PR also contains the changes needed to move the mobile content to its new home.